content/posts/newsletter/2020-05-01.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Draft Title - Date
+save_as: tba
+url: tba
+date: 2020-05-01 00:00
+author: emus
+blog_id: newsletter
+category: Newsletter
+---
+
+# TITLE + DATE
+
+## Intro
+
+Welcome to the XMPP newsletter covering the month of May 2020.
+
+We are always happy about contributors. Just join the discussion in our [Comm-Team group chat (MUC)](xmpp:commteam@muc.xmpp.org?join) and thereby help us sustain this as a community effort. 
```

Comment:
happy about contributors => happy to welcome new contributors

Reply:
Thanks for all the reviews, can you try to create merge requests? So it's easier to integrate your appreciated corrections